CMakeLists.txt

```diff
@@ -1,32 +1,26 @@
-project(pivy)
 cmake_minimum_required(VERSION 3.14)
+project(pivy)
 
-FIND_PACKAGE(SWIG 4.0.0 REQUIRED)
-INCLUDE(${SWIG_USE_FILE})
+find_package(SWIG 4.0.0 REQUIRED)
+include(${SWIG_USE_FILE})
 
-FIND_PACKAGE(Coin CONFIG REQUIRED)
-FIND_PACKAGE(SoQt CONFIG)
+find_package(Coin CONFIG REQUIRED)
```

Comment:
Coin provides *-config.cmake files that can be used with find_package. Those files provide imported targets, that are prefixed with Coin:: or SoQt:: . If Coin and SoQt could be found you should add Coin::Coin and SoQt::SoQt to the  target_link_libraries(...) command.